subsys/bluetooth/host/smp.c

```diff
@@ -4064,6 +4064,7 @@ static uint8_t smp_pairing_failed(struct bt_smp *smp, struct net_buf *buf)
 	return 0;
 }
 
+#if defined(CONFIG_BT_PRIVACY)
 static uint8_t smp_ident_info(struct bt_smp *smp, struct net_buf *buf)
```

Comment:
I don't think we have any specific guidelines for this, but I do have a slight preference of using `IS_ENABLED(CONFIG_BT_PRIVACY)` inside `smp_ident_info` to handle the return value, instead of guarding the entire function. 

Ditto for smp_ident_addr_info

Reply:
I just followed the same pattern as other command handlers have in the file.